src/NServiceBus.Core/Timeout/Hosting/Windows/ManageMessageFailuresWithoutSlr.cs

```diff
@@ -45,7 +47,7 @@ void SendFailureMessage(TransportMessage message, Exception e, string reason)
                 return;
             }
 
-            message.SetExceptionHeaders(e, localAddress ?? config.LocalAddress,reason);
+            message.SetExceptionHeaders(e, failedQueue ?? localAddress ?? config.LocalAddress,reason);
```

Comment:
this reads kinda weird. we're using the `failedQueue` variable over the `localAddress` which will have the value set to `Configure.LocalAddress` ? Sounds more like `localAddress` is initialized incorrectly for satellites?